lib/ngannotate/processor.rb

```diff
@@ -29,10 +29,14 @@ def force
     # Skip processing in environments where it does not make sense.
     # Override by NG_FORCE=true env variable
     #
-    def skip
+    def skip_by_config
       !force && !::Rails.configuration.ng_annotate.process
     end
 
+    def skip
+      skip_by_config || !@file.start_with?(::Rails.root.to_s)
```

Comment:
Tried this and it looks like this is not dealing engines using angularjs in approriate manner. For example, I've myself project setup where host server and engines attached into it use angularjs.